### DISCUSSION TITLE
Bug? CanonSmiles with IsotopeParent gives different SMILES

### DISCUSSION THREAD

**Discussion by rgiessmann** [rdkit/rdkit#6878] [Q&A] [answered]
Hi there! 

I am standardizing molecules to be independent of their isotopes, using rdkit.Chem.MolStandardize.rdMolStandardize.IsotopeParent

I also came across rdkit.Chem.CanonSmiles and thought that would be a nice way to store the representations of the Mols on disk.

When I ran "CCO" vs. "[13C]CCO" through the IsotopeParent and then did rdkit.Chem.MolToSmiles , CanonSmiles returned "[C]CO" for the molecules having had a 13C before, whereas it returned "CCO" for the normal one.

Is this a bug? Can you reproduce? I am not sure what I found here.

Thanks for all the work and best!
    Robert

**Comment by dehaenw** [comment, marked as answer]
in smiles when you use bracket notation you need to explicitly provide hydrogen count. so the molecule you intend is `[13CH3]CO` and not `[13C]CO`

**Reply by rgiessmann** [reply]
True! I should have checked a drawing of the molecule :( Didn't know about hydrogen count with the brackets. Thanks @dehaenw !
